packages/python/cisco_opentelemetry_specifications/payload_attributes.py

```diff
@@ -34,4 +34,6 @@
     "messaging.rabbitmq.payload",
     "db.redis.arguments",
     "db.redis.response",
+    "db.mongodb.arguments",
```

Comment:
If arguments means query, there's an OTel semantic for that which called DB_STATEMENT